config/servicemappings/edgecontainer.yaml

```diff
@@ -0,0 +1,136 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: core.cnrm.cloud.google.com/v1alpha1
+kind: ServiceMapping
+metadata:
+  name: edgecontainer.cnrm.cloud.google.com
+  namespace: cnrm-system
+spec:
+  name: Edgecontainer
+  version: v1beta1
+  serviceHostName: edgecontainer.googleapis.com
+  resources:
+    - name: google_edgecontainer_cluster
+      kind: EdgecontainerCluster
+      idTemplate: "projects/{{project}}/locations/{{location}}/clusters/{{name}}"
+      idTemplateCanBeUsedToMatchResourceName: false
+      resourceAvailableInAssetInventory: false
+      metadataMapping:
+        name: name
+        labels: labels
+      resourceID:
+        targetField: name
+      hierarchicalReferences:
+        - type: project
+          key: projectRef
+      resourceReferences:
+        - tfField: authorization.admin_users.username
+          key: serviceAccountRef
+          gvk:
+            kind: IAMServiceAccount
+            version: v1beta1
+            group: iam.cnrm.cloud.google.com
+          targetField: email
+        - tfField: control_plane_encryption.kms_key
+          key: bootDiskKMSCryptoKeyRef
+          gvk:
+            kind: KMSCryptoKey
+            version: v1beta1
+            group: kms.cnrm.cloud.google.com
+          targetField: self_link
+        - tfField: project
+          key: projectRef
+          description: |-
+            The project that this resource belongs to.
+          gvk:
+            kind: Project
+            version: v1beta1
+            group: resourcemanager.cnrm.cloud.google.com
+        - tfField: fleet.project
+          key: projectRef
+          description: |-
+            The number of the Fleet host project where this cluster will be registered.
+          gvk:
+            kind: Project
+            version: v1beta1
+            group: resourcemanager.cnrm.cloud.google.com
+          targetField: number
+          valueTemplate: "projects/{{value}}"
+    - name: google_edgecontainer_node_pool
+      kind: EdgecontainerNodePool
+      autoGenerated: true
```

Comment:
Please remove `autoGenerated: true`, this resource is not auto-generated.

Reply:
Done.